runtime/compiler/control/rossa.cpp

```diff
@@ -879,7 +879,7 @@ internalCompileClass(J9VMThread * vmThread, J9Class * clazz)
          {
          bool queued = false;
          TR_MethodEvent event;
-         event._eventType = TR_MethodEvent::InterpreterCounterTripped;
+         event._eventType = TR_MethodEvent::CompileMethodAPI;
```

Comment:
I see that in this case we set 
`plan->setIsExplicitCompilation(true);`
in line 891, below. I am thinking that flag is enough to detect these types of compilations. `launchGPU()` also sets `plan->setIsExplicitCompilation(true)`  and I think those compilations do not use a invocation count as well.
I don't see who is using that information; maybe the code that made use of that information has been deleted/cleaned-up.